mgr/src/server/services/fix_layout.cr

```diff
@@ -10,7 +10,7 @@ class FixLayoutService < Service
     @create_pid_file = true
 
     @path = Path[PROGRAM_NAME].expand.to_s
-    @id = "#{volume_name}-fix-layout"
+    @id = "rebalance-fix-layout-#{storage_unit.path}"
```

Comment:
Use `gsub` here instead of adding it in other places since storage unit path can have slashes. That causes multiple subdirs in PID file and other places

Reply:
Ack

Reply:
done